core-services/release-controller/_releases/priv/release-ocp-4.11.json

```diff
@@ -76,6 +76,13 @@
                 "name": "periodic-ci-openshift-release-master-nightly-4.11-e2e-aws-fips-priv"
             }
         },
+        "aws-imdsv2": {
+            "disabled": true,
```

Comment:
Why disabled?

Reply:
Who knows. Purposes of this file and directory are a bit cryptic for me. This entry was generated by `make release-controllers`.

Reply:
All entries there have "disabled": true